pkg/project/v2/project_loader_test.go

```diff
@@ -108,155 +107,132 @@ func Test_findDuplicatedConfigIdentifiers(t *testing.T) {
-	_ = afero.WriteFile(testFs, "project/not-dashboard-dir/board.json", []byte("{}"), 0644)
+	testFs := testutils.TempFs(t)
+	require.NoError(t, testFs.MkdirAll("project/alerting-profile", 0755))
+	require.NoError(t, testFs.MkdirAll("project/not-dashboard-dir", 0755))
+	require.NoError(t, afero.WriteFile(testFs, "project/alerting-profile/profile.yaml", []byte("configs:\n- id: profile\n  config:\n    name: Test Profile\n    template: profile.json\n  type:\n    api: alerting-profile"), 0644))
+	require.NoError(t, afero.WriteFile(testFs, "project/alerting-profile/profile.json", []byte("{}"), 0644))
+	require.NoError(t, afero.WriteFile(testFs, "project/not-dashboard-dir/board.yaml", []byte("configs:\n- id: board\n  config:\n    name: Test Dashboard\n    template: board.json\n  type:\n    api: dashboard"), 0644))
+	require.NoError(t, afero.WriteFile(testFs, "project/not-dashboard-dir/board.json", []byte("{}"), 0644))
 
 	context := getSimpleProjectLoaderContext([]string{"project"})
 
 	got, gotErrs := LoadProjects(testFs, context)
+	assert.Len(t, gotErrs, 0, "Expected to load project without error")
+	assert.Len(t, got, 1, "Expected a single loaded project")
 
-	assert.Equal(t, len(gotErrs), 0, "Expected to load project without error")
-	assert.Equal(t, len(got), 1, "Expected a single loaded project")
-
-	c, found := got[0].Configs["env"]
-	assert.Assert(t, found, "Expected configs loaded for test environment")
-	assert.Equal(t, len(c), 2, "Expected a dashboard and alerting-profile configs in loaded project")
+	db := findConfigs(t, got[0], "env", "dashboard")
+	assert.Len(t, db, 1, "Expected a one config to be loaded for dashboard")
 
-	db, found := c["dashboard"]
-	assert.Assert(t, found, "Expected configs loaded for dashboard api")
-	assert.Equal(t, len(db), 1, "Expected a one config to be loaded for dashboard")
-
-	a, found := c["alerting-profile"]
-	assert.Assert(t, found, "Expected configs loaded for dashboard api")
-	assert.Equal(t, len(a), 1, "Expected a one config to be loaded for alerting-profile")
+	a := findConfigs(t, got[0], "env", "alerting-profile")
+	assert.Len(t, a, 1, "Expected a one config to be loaded for alerting-profile")
 }
 
 func TestLoadProjects_LoadsProjectInRootDir(t *testing.T) {
-	testFs := afero.NewMemMapFs()
-	_ = afero.WriteFile(testFs, "project/profile.yaml", []byte("configs:\n- id: profile\n  config:\n    name: Test Profile\n    template: profile.json\n  type:\n    api: alerting-profile"), 0644)
-	_ = afero.WriteFile(testFs, "project/profile.json", []byte("{}"), 0644)
-	_ = afero.WriteFile(testFs, "project/board.yaml", []byte("configs:\n- id: board\n  config:\n    name: Test Dashboard\n    template: board.json\n  type:\n    api: dashboard"), 0644)
-	_ = afero.WriteFile(testFs, "project/board.json", []byte("{}"), 0644)
+	testFs := testutils.TempFs(t)
```

Comment:
Quick note: in case `filepath.Abs` is involved (like in integration tests or in monaco deploy), `testutils.TempFs(t)` doesn't work with as it shifts/reduces the file system, while `filepath.Abs` deals with actual FS.